src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/AdditionalHearingDocumentService.java

```diff
@@ -227,10 +229,17 @@ public void bulkPrintAdditionalHearingDocuments(CaseDetails caseDetails, String
             document.add(additionalUploadedDoc);
         }
 
-        BulkPrintDocument additionalDoc
-            = documentHelper.getBulkPrintDocumentFromCaseDocument(additionalHearingDocument.getAdditionalHearingDocument().getDocument());
-
-        document.add(additionalDoc);
+        if (!CollectionUtils.isEmpty(additionalHearingDocumentData)) {
```

Comment:
It think it is optional field. Throwing error on this will break the flow.
Should not throw error.